webpack/react_app/components/TargetingHosts/__tests__/fixtures.js

```diff
@@ -37,7 +37,29 @@ export const TargetingHostsFixtures = {
     ],
   },
   'renders with loading': {
-    status: '',
+    apiStatus: 'PENDING',
     items: [],
   },
 };
+
+export const TargetingHostsPageFixtures = {
+  renders: {
+    handleSearch: () => {},
+    searchQuery: '',
+    apiStatus: 'RESOLVED',
+    items: [
+      {
+        name: 'host',
+        link: '/link',
+        status: 'success',
+        actions: [],
+      },
+    ],
```

Comment:
I think you can re-use this object in all of the fixtures, just declare it on the top of the page.